SECURITY.md

```diff
@@ -1,3 +1,8 @@
+<!--
+Copyright The Shipwright Contributors
+
+SPDX-License-Identifier: Apache-2.0
+-->
```

Comment:
Unsure here as well, but I could understand that we leave this one here in.

Reply:
`README.md` have this one also, therefore I stick to that approach.